plugins/leemons-plugin-users/frontend/src/pages/private/users/DetailUser/UserImage.js

```diff
@@ -37,8 +38,8 @@ const Styles = createStyles((theme) => ({
 function UserImage({ t, user, session, form, isEditMode }) {
   const isMe = user.id === session?.id;
   const [, , , getErrorMessage] = useRequestErrorMessage();
+  const [uploadingFileInfo, setUploadingFileInfo] = useState(null);
   const { classes: styles } = Styles({}, { name: 'UserImage' });
-  const { setLoading } = useLayout();
```

Comment:
I guess you removed setLoading because it shows a modal of uploading file, right?

Reply:
I removed it because it was only used when making the change avatar request. For this request we want show the uploading modal instead of the loading wheel.